src/flatpakManifestUtils.ts

```diff
@@ -136,3 +136,15 @@ function isValidManifest(manifest: FlatpakManifestSchema): boolean {
     const hasModules = manifest.modules !== undefined
     return hasId && hasModules
 }
+
+
+/**
+ * Check if version1 is newer or equal than version2
+ * @param version1 a flatpak version, usually returned by flatpak --version
+ * @param version2 a flatpak version, required by the manifest
+ * @returns Whether version1 is newer or equal than version2
+ */
+export function versionCompare(version1: string, version2?: string): boolean {
```

Comment:
This could maybe renamed to `isValidVersion` or something more appropriate, then handle the `flatpak --version` call inside, the downside is it cannot be tested

Reply:
Well, my thinking is once your PR for #8 lands, we want to not allow any commands to be executed and not only show an error message. So the version has to be stored somewhere once instead of calling that everytime

Reply:
It could be a static variable like IS_SANDBOXED then use it within the function, if the flatpak version doesn't change on the entire vscode session

Reply:
Makes sense

Reply:
The issue with doing that though is we can no longer test the function which is something I would like to keep in case we notice the comparison logic is too flawed and we would need a different implementation

Reply:
there could be a helper version compare function in utils.ts that can be tested which then called by `isValidVersion`  (or named something clearer)

Reply:
It is only useful for comparing the flatpak version, so let us keep it as is and see if it is worth changing in the future